modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shibari_worn.dm

```diff
@@ -0,0 +1,333 @@
+
```

Comment:
to be honest this could afford to be split up into multiple files in its own folder